Sources/WebDriver/Session.swift

```diff
@@ -282,15 +282,6 @@ public class Session {
             session: id, element: element?.id, xOffset: xOffset, yOffset: yOffset))
     }
 
-    /// Sends key presses to this session.
-    /// - Parameter keys: An array of key sequences according to the WebDriver spec.
-    /// - Parameter releaseModifiers: A boolean indicating whether to release modifier keys at the end of the sequence.
-    public func sendKeys(_ keys: [Keys], releaseModifiers: Bool = true) throws {
```

Comment:
You'll also need to update:
https://github.com/thebrowsercompany/swift-webdriver/blob/4d658fdf701c190c4a8dc0957d3c8538d305fb70/Docs/SupportedAPIs.md?plain=1#L43

Reply:
Why? It's still supported, just not that overload

Reply:
Well if `Session.sendKeys()` is deleted, we should remove the reference to it.  Maybe I didn't read it right?

Reply:
Check your prescription :P
![image](https://github.com/user-attachments/assets/86df9e0c-38d6-4548-af07-49d8f9d4c184)

Reply:
Fair.